composer.json

```diff
@@ -7,15 +7,15 @@
     "config": {
         "bin-dir": "./bin",
         "platform": {
-            "php": "7.1.33",
+            "php": "7.3.11",
             "ext-dom": "0.0.0",
             "ext-mbstring": "0.0.0"
         },
         "sort-packages": true
     },
     "description": "Standalone Solid Server written in PHP by PDS Interop.",
     "license": "MIT",
-    "name": "pdsinterop/solid-server",
+    "name": "pdsinterop/php-solid-server",
```

Comment:
The name is in the contaxt of PHP packages, so unless there is a compelling reason, the `php` is not needed here.